src/libcfitsio.jl

```diff
@@ -208,7 +208,7 @@ function fits_create_file(filename::AbstractString)
     ptr = Array(Ptr{Void}, 1)
     status = Cint[0]
     ccall((:ffinit,libcfitsio), Cint, (Ptr{Ptr{Void}},Ptr{UInt8},Ptr{Cint}),
-          ptr, bytestring(filename), status)
+          ptr, string(filename), status)
```

Comment:
Should all these cases of `bytestring(x)` just be replaced by `x`? `string(x::String) === x` on 0.5 and `bytestring(x::ASCIIString) === x` on 0.4. Was this necessary on 0.3? I'm having a hard time remembering...